sktime/transformers/sktime.transformers.series.acf.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# import numpy as np
+# import pandas as pd
+
+# from statsmodels.tsa.stattools import acf
+from sktime.transformers.base import _SeriesToSeriesTransformer
+from sktime.utils.validation.series import check_series
+
+
+class AutoCorrelationFunctionTransformer(_SeriesToSeriesTransformer):
+    def transform(
+        self,
+        x,
+        unbiased=False,
```

Comment:
Make sure you have the same interface here as in the abstract base class. I'd consider the additional arguments here as hyper-parameters and initialise them in `__init__`